docs/topics/mta-web-subv-rep.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-subv-rep_{context}"]
+= Configuring Subversion repositories
+
+
+.Procedure
+
+. In the *Administrator* view, click *Repositories* and then *Subversion*.
+. Toggle *Consume insecure Git repositories* to enable.
```

Comment:
To enable what? "Enable" is a transitive verb, so it requires a direct object.

Reply:
Toggle Consume insecure Git repositories to enable the configuration.